libs/output-mapping/phpunit.xml.dist

```diff
@@ -10,13 +10,34 @@
          syntaxCheck="false"
          bootstrap="bootstrap.php">
 
-    <testsuite name="main">
-        <directory>tests</directory>
-        <exclude>
-            <file>tests/Writer/TableDefinitionTest.php</file>
-        </exclude>
+    <testsuite name="unit-tests">
+        <directory>tests/Configuration</directory>
+        <directory>tests/DeferredTasks</directory>
+        <directory>tests/Staging</directory>
+        <directory>tests/Table</directory>
+        <directory>tests/Writer/File</directory>
+        <directory>tests/Writer/Table</directory>
+        <directory>tests/Writer/Helper</directory>
+    </testsuite>
+    <testsuite name="main-writer-tests-1">
+        <file>tests/Writer/StorageApiLocalTableWriterTest.php</file>
+    </testsuite>
+    <testsuite name="main-writer-tests-2">
+        <directory>tests/Writer</directory>
+        <exclude>tests/Writer/File</exclude>
+        <exclude>tests/Writer/Helper</exclude>
+        <exclude>tests/Writer/Table</exclude>
+        <exclude>tests/Writer/Redshift</exclude>
+        <exclude>tests/Writer/Workspace</exclude>
+        <exclude>tests/Writer/TableDefinitionTest.php</exclude>
```

Comment:
ten se někde spouští TableDefinitionTest ?

Reply:
jop, testsuite "native-types"

Reply:
Asi bych byl za to do budoucna udelat treba slozku per test suite. Vim ze Odin asi nebude happy, protoze se to nemapuje 1:1 a IDE s tim muze mit problemy na proklik kod<->test :slightly_smiling_face: , ale prijde mi, ze tohle se rozbije s druhym pridanym testem :smile: